lib/galaxy/jobs/runners/__init__.py

```diff
@@ -129,7 +129,13 @@ def run_next(self):
             except Exception:
                 name = 'unknown'
             try:
+                action_str = 'galaxy.jobs.runners.%s.%s' % (self.__class__.__name__.lower(), name)
+                action_timer = self.app.execution_timer_factory.get_timer(
+                    'internals.%s' % action_str,
+                    'job runner action %s for job ${job_id} executed' % (action_str)
+                )
                 method(arg)
+                log.trace(action_timer.to_str(job_id=job_id))
```

Comment:
so if one wants to see if there are any performance degradation introduced by a PR, then they would need to scan/parse the logs?